lib/reportsv2.rb

```diff
@@ -75,8 +75,8 @@ def initialize(opts={})
     def report(type, extension, params)
       raise "workspace_id is required" if @workspace_id.nil?
       get "#{type}#{extension}", {
-        'user_agent': @user_agent,
-        'workspace_id': @workspace_id,
+        :'user_agent' => @user_agent,
```

Comment:
Please remove the leading colons. I had it backwards in my earlier comments. The key should be a string, not a symbol. (I definitely was just using Python syntax in Ruby.)

So this line, for example, should be: `'user_agent' => @user_agent,`